integration-tests/tests/hacbs/add-component-to-application-via-hacbs-tab.spec.ts

```diff
@@ -1,136 +1,166 @@
+import { IntegrationTestsTabPage } from '../../support/pages/hacbs/tabs/IntegrationTestsTabPage';
 
 describe('Create Components using the HACBS UI', () => {
-    const LOCAL_STORAGE_KEY_GS_MODAL = 'hacbs/getting-started-modal';
-    const LOCAL_STORAGE_KEY_QUICKSTART = 'hacbs/showApplicationQuickstart';
-    const LOCAL_STORAGE_KEY_APPLICATION_MODAL = 'hacbs/showApplicationModal';
-    const applicationName = Common.generateAppName();
-    const integrationTestsTabPage = new IntegrationTestsTabPage();
-    const containerImage = 'https://quay.io/kpavic/test-bundle:pipeline';
-    const pipelineName = 'demo-pipeline';
-    const publicRepos = [
-            'https://github.com/dheerajodha/devfile-sample-code-with-quarkus',
-            'https://github.com/devfile-samples/devfile-sample-go-basic.git',
-            'https://github.com/nodeshift-starters/devfile-sample.git'
-            ]
-    const componentNames = ['java-quarkus', 'go', 'go-component', 'nodejs'];
-    const integrationTestNames = ['my-test-1', 'my-test-2', 'my-test-3', 'my-test-4', 'my-optional-test'];
-    const integrationTestMetadata = [integrationTestNames[4], 'quay.io/kpavic/test-bundle:pipeline', pipelineName, 'test.appstudio.openshift.io/optional=true', applicationName];
-
-    before(function () {
-        // Enable HACBS
-        localStorage.setItem('hacbs', 'true');
-        localStorage.setItem(LOCAL_STORAGE_KEY_GS_MODAL, 'true');
-        localStorage.setItem(LOCAL_STORAGE_KEY_APPLICATION_MODAL, 'true');
-        // Need to reload the page after enabling HACBS via localStorage
-        cy.reload();
+  const LOCAL_STORAGE_KEY_GS_MODAL = 'hacbs/getting-started-modal';
+  const LOCAL_STORAGE_KEY_QUICKSTART = 'hacbs/showApplicationQuickstart';
+  const LOCAL_STORAGE_KEY_APPLICATION_MODAL = 'hacbs/showApplicationModal';
+  const applicationName = Common.generateAppName();
+  const integrationTestsTabPage = new IntegrationTestsTabPage();
+  const containerImage = 'https://quay.io/kpavic/test-bundle:pipeline';
+  const pipelineName = 'demo-pipeline';
+  const publicRepos = [
+    'https://github.com/dheerajodha/devfile-sample-code-with-quarkus',
+    'https://github.com/devfile-samples/devfile-sample-go-basic.git',
+    'https://github.com/nodeshift-starters/devfile-sample.git',
+  ];
+  const componentNames = ['java-quarkus', 'go', 'go-component', 'nodejs'];
+  const integrationTestNames = [
+    'my-test-1',
+    'my-test-2',
+    'my-test-3',
+    'my-test-4',
+    'my-optional-test',
+  ];
+  const integrationTestMetadata = [
+    integrationTestNames[4],
+    'quay.io/kpavic/test-bundle:pipeline',
+    pipelineName,
+    'test.appstudio.openshift.io/optional=true',
+    applicationName,
+  ];
+
+  before(function () {
+    localStorage.setItem(LOCAL_STORAGE_KEY_GS_MODAL, 'true');
+    localStorage.setItem(LOCAL_STORAGE_KEY_APPLICATION_MODAL, 'true');
+    // Need to reload the page after enabling HACBS via localStorage
+    cy.reload();
```

Comment:
Likely don't need this anymore but do need to reload the non-hacbs tests.

Reply:
Updated